app/controllers/api/users_controller.rb

```diff
@@ -7,11 +7,14 @@ def create
     begin
       saved = user.save!
     rescue ActiveRecord::StatementInvalid => invalid
+      flash[:error] = "Attorney failed to create!";
```

Comment:
yo there is a slight inconsistency here since in all previous instances of unsuccessful flashes you don't use an exclamation mark but here you decide to use one??? unmergeable until this is addressed tbh